app/models/order.rb

```diff
@@ -5,12 +5,23 @@ class Order <ApplicationRecord
   has_many :item_orders
   has_many :items, through: :item_orders
 
+  def self.orders_sorted_for_admin_display
+    sorted_orders = []
+    sorted_orders << Order.where("status = 'Packaged'")
+    sorted_orders << Order.where("status = 'Pending'")
+    sorted_orders << Order.where("status = 'Shipped'")
+    sorted_orders << Order.where("status = 'Cancelled'")
+    sorted_orders.flatten
+  end
```

Comment:
I suspect there's a more elegant way to do this (possibly through SQL injecting a Union?) but for now this allows us to pass a properly sorted set of orders to the Admin dashboard.

Reply:
Yeah, I'm not quite sure the alternative off the top of my head

Reply:
SELECT * FROM order GROUP_BY status in SQL? Would love to revisit this.